dms-persistence/src/main/kotlin/team/aliens/dms/persistence/outing/repository/vo/QueryStudentOutingHistoryVO.kt

```diff
@@ -0,0 +1,21 @@
+package team.aliens.dms.persistence.outing.repository.vo
+
+import com.querydsl.core.annotations.QueryProjection
+import team.aliens.dms.domain.outing.spi.vo.OutingCompanionDetailsVO
+import java.util.UUID
+
+class QueryStudentOutingHistoryVO @QueryProjection constructor(

```

Comment:
```suggestion
class QueryOutingCompanionDetailsVO @QueryProjection constructor(
```
이런식으로 다른 VO 네이밍과 통일하면 좋을 것 같아요.